### PR TITLE
Add secrets to environment for publishing.

### PR DESCRIPTION
**Pull request by anuraaga**
Forgot this

### CODE REVIEW

**Review by jkwatson — APPROVED** [review]

File: .github/workflows/continuous-build.yml [hunk 1]
```diff
@@ -65,3 +65,6 @@ jobs:
           remote-build-cache-proxy-enabled: false
           # TODO(anuraaga): Remove version specifier after next release creates a tag on master.
           arguments: artifactoryPublish -Prelease.version=0.10.0-SNAPSHOT
+        env:
+          BINTRAY_USER: ${{ secrets.BINTRAY_USER }}
+          BINTRAY_KEY: ${{secrets.BINTRAY_KEY }}
```
Comment (jkwatson) [inline .github/workflows/continuous-build.yml:70]: does the variance in whitespace make any difference here?

Comment (anuraaga) [reply .github/workflows/continuous-build.yml:70]: Which whitespace? Are you referring to the difference in the length of the word USER vs KEY? :P

Comment (jkwatson) [reply .github/workflows/continuous-build.yml:70]:
```suggestion
          BINTRAY_KEY: ${{ secrets.BINTRAY_KEY }}
```

Comment (anuraaga) [reply .github/workflows/continuous-build.yml]: Thanks my eyes just could not find that (no coffee yet is a problem)

**Pull request merged**